scripts/azureml-assets/azureml/assets/deployment_config.py

```diff
@@ -58,7 +58,7 @@ class AssetTags:
     """
 
     add: Dict[str, str] = None
-    replace: Dict[str, str] = None
+    replace: Dict[str, Union[list, str]] = None

```

Comment:
Please revert this. Tag values are strings. If any other datatypes need to be handled, conversion should be upstream.

Reply:
Done